jwst/wfss_contam/observations.py

```diff
@@ -186,9 +186,9 @@ def disperse_all(self, order, wmin, wmax, sens_waves, sens_resp, cache=False):
                 self.cached_object[i]['maxy'] = []
 
             # Disperse object "i"
-            self.disperse_chunk(i, order, wmin, wmax, sens_waves, sens_resp)
+            self.disperse_chunk(i, order, wmin, wmax, sens_waves, sens_resp, [0, 0])
```

Comment:
The offsets for this bit of code are the part I'm not sure of - it looks like prior to my changes, it was going the same route as source-by-source dispersion, so perhaps the proper offsets need to be passed down this chain for each source cutout as well? @hbushouse